frontend/src/styles/utilities.scss

```diff
@@ -690,3 +693,7 @@
         list-style-type: #{$list_style_type};
     }
 }
+
+.shadow {
+    box-shadow: 0 16px 16px -16px rgba(0, 0, 0, 0.35);
```

Comment:
Could use `var(--shadow-elevation)` just in case

Reply:
Ah good catch